Refureku/Library/Include/Internal/Refureku/TypeInfo/Variables/VariableBaseImpl.h

```diff
@@ -31,6 +37,13 @@ namespace rfk
 			*	@return _type.
 			*/
 			inline Type const& getType() const noexcept;
+
+			/**
+			*	@brief Getter for the field _canonicalTypeName.
+			* 
+			*	@return _canonicalTypeName.
+			*/
+			inline const char* getCanonicalTypeName() const noexcept;
```

Comment:
The project uses east const nomenclature so change to `char const*` for consistency.